src/components/graph-view.js

```diff
@@ -904,10 +902,15 @@ class GraphView extends Component {
   renderView = () => {
     var nodes = this.props.nodes;
     var edges = this.props.edges;
-
     // Update the view w/ new zoom/pan
-    const view = d3.select(this.view)
-      .attr("transform", this.state.viewTransform);
+    const view = d3.select(this.view);
+    const viewNode = view.node();
+    if (viewNode) {
+      const { k, x, y } = this.state.viewTransform;
+      view.attr("transform", this.state.viewTransform);
+      // viewNode.setAttribute('transform', `translate(${x},${y}) scale(${k})`);
```

Comment:
This is a note to understand what d3 does with transform. We could do the same thing with raw JS if we wanted.